experimental/adaptive-dialog/README.md

```diff
@@ -44,6 +44,14 @@ You can report any issues you find or feature suggestions on our GitHub reposito
 You can use this [Visual studio code debugger extension][18] to debug both code based as well as declaratively defined Adaptive Dialogs.
 
 ## Change Log
+### 4.7 PREVIEW
+- \[**New\] Adaptive dialogs have been merged into `botbuilder-dotnet` master branch and now is built on top of the core SDK.
+- \[**New\] ***BREAKING CHANGE*** Renamed `Steps` -> `Actions`, `Rules` -> `Triggers`. Updated all triggers to follow `OnXXX` notation. 
```

Comment:
There are few breaking changes. DO you have a list of breaking canges. Maybe just link to it?

Reply:
Added a list of breaking changes to readme.md